rep-0003.rst

```diff
@@ -233,6 +233,66 @@ Buildsystem support:
 
   - build from source
 
+Lunar Loggerhead (May 2017 - May 2019)
+----------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 25
+
+Architecture supported(CI enabled for):
+- amd64
+- arm32
+- arm64
+Not building x86 packages anymore
```

Comment:
I would suggest not listing it here at all, but have it discussed below. It's useful for those considering migrating. But as a reference after we've settled it, it just won't be on the list.